python/paddle/fluid/framework.py

```diff
@@ -2756,6 +2756,15 @@ class Program(object):
     it will contains nested block.
     Please reference the framework.proto for details.
```

Comment:
reference->refer to

Reply:
Changed, but these are not related to my PR